weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTable.tsx

```diff
@@ -1084,99 +1154,264 @@ export const CallsTable: FC<{
           </div>
         </TailwindContents>
       }>
-      {isMetricsChecked && (
-        <CallsCharts
-          entity={entity}
-          project={project}
-          filter={filter}
-          filterModelProp={filterModelResolved}
+      <style>
```

Comment:
this style block seems unnecessary: 
1. redundant with other style inlining
2. everything can be put directly into the style blocks. 

We already mix direct `style` tags, MUI's `sx` tag, and Tailwind, it would be nice to not have another style method layered in